.github/workflows/build_steps.yml

```diff
@@ -231,10 +231,9 @@ jobs:
           disk-root: "D:"  # This is also the checkout directory. Total size 12GB.
         continue-on-error: true
 
+
       - name: Install azurite
-        if: matrix.os == 'linux'
-        run: |
-          yum install -y npm
+        uses: actions/setup-node@v3
           npm install -g azurite
```

Comment:
The syntax is a bit broken: `uses:` calls an external action, so you will need a separate

```
- name: ...
  run: npm install -g azurite
```

However, AFAIK, all GitHub runner images have npm pre-installed. You don't need to do the `setup-node` (or `yum install`)

Reply:
Seems the linux image we used does not come with npm